nomad/structs/structs.go

```diff
@@ -11114,6 +11114,13 @@ type Allocation struct {
 	// AllocatedResources is the total resources allocated for the task group.
 	AllocatedResources *AllocatedResources
 
+	// HostVolumeIDs is a list of host volume IDs that this allocation
+	// requires.
+	HostVolumeIDs []string
+
+	// CSIVolumeIDs is a list of CSI volume IDs that this allocation requires.
+	CSIVolumeIDs []string
```

Comment:
Clarifies that this is the volumes it's actually been given, not the ones it's asking for:

```suggestion
	// HostVolumeIDs is a list of host volume IDs that this allocation
	// has claimed.
	HostVolumeIDs []string

	// CSIVolumeIDs is a list of CSI volume IDs that this allocation has claimed.
	CSIVolumeIDs []string
```